fhir_river_api/requirements.txt

```diff
@@ -0,0 +1,4 @@
+requests==2.21.0
```

Comment:
I don't think you use `requests`, do you?

Reply:
I don't indeed 👌